src/features/asciidoctorConfig.ts

```diff
@@ -17,9 +17,19 @@ export async function getAsciidoctorConfigContent (documentUri: vscode.Uri): Pro
   if (workspaceFolder === undefined) {
     return undefined
   }
-  const asciidoctorConfigUri = vscode.Uri.joinPath(workspaceFolder.uri, '.asciidoctorconfig')
+  const configContent = await getConfigContent(workspaceFolder, '.asciidoctorconfig')
```

Comment:
I might be wrong but it seems that the Intellij plugin will return the content of both files. So if you have:

**.asciidoctorconfig**
```
:name: Aurélien
:version: 1.2.3
```
**.asciidoctorconfig.adoc**
```
:name: Alexander
```

It will include the following contents:

```
:name: Aurélien
:version: 1.2.3


:name: Alexander


```

https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/64707c7632e1c5e6efc34100dc3cb2dd0bab4ea6/src/main/java/org/asciidoc/intellij/AsciiDocWrapper.java#L710-L745

So `.asciidoctorconfig.adoc` has higher precedence and both files are taken into account.

Reply:
@ahus1 is that correct?

Reply:
@Mogztter - yes, you're right, this is the order it is being processed in. 

I doubt that a user should have both files in a folder, still it makes sense to have a consistent behavior across tools.

Thanks for paying extra attention here!

Reply:
Thanks for pointing it out. I updated the Pull Request to have the same behavior